FirebaseAuth/Tests/Unit/PhoneAuthProviderTests.swift

```diff
@@ -546,24 +545,21 @@
                                     bothClientAndAppID: Bool = false,
                                     reCAPTCHAfallback: Bool = false,
                                     forwardingNotification: Bool = true,
-                                    presenterError: Error? = nil) throws {
+                                    presenterError: Error? = nil) async throws {
       initApp(function, useClientID: useClientID, bothClientAndAppID: bothClientAndAppID,
               testMode: testMode,
               forwardingNotification: forwardingNotification)
       let auth = try XCTUnwrap(PhoneAuthProviderTests.auth)
       let provider = PhoneAuthProvider.provider(auth: auth)
-      let expectation = self.expectation(description: function)
 
       if !reCAPTCHAfallback {
         // Fake out appCredentialManager flow.
         auth.appCredentialManager?.credential = AuthAppCredential(receipt: kTestReceipt,
                                                                   secret: kTestSecret)
       } else {
         // 1. Intercept, handle, and test the projectConfiguration RPC calls.
-        let projectConfigExpectation = self.expectation(description: "projectConfiguration")
```

Comment:
This one is still needed unless an `async` alternative is used for `projectConfigRequester`